lib/WebLedgerClient.js

```diff
@@ -125,29 +130,34 @@ class WebLedgerClient {
    */
   async getDocument({headers = {}, service}) {
     const baseURL = await this.getServiceEndpoint({serviceId: service});
-    const serviceApi =
-      create({baseURL, httpsAgent: this.httpsAgent, timeout: this.timeout});
-    const requestHeaders = Object.assign(
-      {}, constants.DEFAULT_HEADERS, headers);
-    const response = await serviceApi.get('/', {}, {
-      headers: requestHeaders
-    });
-    if(response.problem) {
+    try {
+      const response = await httpClient.get('', {
+        prefixUrl: baseURL,
+        agent: this.httpsAgent,
+        timeout: this.timeout,
+        headers: {...DEFAULT_HEADERS, ...headers}
+      });
```

Comment:
While this `prefixUrl` is cool, I think sticking with the same construction as on L68 is better: `httpClient.get(baseUrl, {...})`

Reply:
the reason for `prefixUrl` is that it closest mimics `apisauce`'s `create` method. In other words `prefixUrl` saved a lot of work here. Maybe a comment above `prefixUrl` might be better?